### PR TITLE
Copter: Add MANUAL_CONTROL to telemetry

### PR DESCRIPTION
**Pull request by muramura**
I want to get the value of the propo as a joystick value.
I can operate the vehicle with MANUAL_CONTROL by getting this value.
The advantage is that the CC can operate the vehicle without being aware of the PWM value.

RC1-4: 1000 PWM
Screenshot from 2022-12-06 02-44-30

RC1-4: 1500 PWM
Screenshot from 2022-12-06 02-46-49

RC1-4: 2000 PWM
Screenshot from 2022-12-06 02-47-34

### CODE REVIEW

File: ArduCopter/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -1473,6 +1479,18 @@ void GCS_MAVLINK_Copter::send_wind() const
         wind.z);
 }
 
+void GCS_MAVLINK_Copter::send_manual_control() const
+{
+    mavlink_msg_manual_control_send(
+        chan,
+        1,
```
Comment (WickedShell) [inline ArduCopter/GCS_Mavlink.cpp:1486]: Needs to be the GCS id, not hard coded. In fact by default this actually would target the same autopilot that sends the message, particularly bad if you have any routing loops.

File: ArduCopter/GCS_Mavlink.cpp [hunk 2]
```diff
@@ -508,6 +513,7 @@ static const ap_message STREAM_RC_CHANNELS_msgs[] = {
     MSG_SERVO_OUTPUT_RAW,
     MSG_RC_CHANNELS,
     MSG_RC_CHANNELS_RAW, // only sent on a mavlink1 connection
+    MSG_MANUAL_CONTROL,
```
Comment (WickedShell) [inline ArduCopter/GCS_Mavlink.cpp:516]: There's too many things in this stream. I'm not sure if we should be sending this by default, and leave it up to the stream rate request handling.

**Review by WickedShell — CHANGES_REQUESTED** [review]
Why does this help a companion computer? Normalized inputs should be possible to be used directly.

I do like the idea of normalized monitoring, but I'm not sure that I am seeing how this helps us in the application, particularly enough to justify the extra bandwidth usage by default.

**Comment by peterbarker** [comment]
This is a message used to control vehicles - it's right there in the `description` field.

It's *not* a reporting message.  If we wanted one of those we'd make one - in the same way that we have "RC_CHANNELS".  Alternatively, you could try to convince upstream mavlink to change the description of this message to also allow for reporting - but I think such re-use would be a terrible idea when a new message would be relatively straight-forward.

Sorry, wrong use of this message, closing this.

**Pull request closed without merging**